py-genai/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  python-genai:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8081"
+    environment:
+      - PORT=8081
```

Comment:
Specifying PORT everywhere is unnecessary, if we map the ports in ways, that don't conflict. Just use the defaults and map them out.